calyx-py/calyx/builder.py

```diff
@@ -1101,9 +1101,9 @@ def as_control(obj):
             "GroupBuilder represents continuous assignments and"
             " cannot be used as a control statement"
         )
-        assert not isinstance(
-            gl, ast.CombGroup
-        ), "Cannot use combinational group as control statement"
+        assert not isinstance(gl, ast.CombGroup), (
+            "Cannot use combinational group as control statement"
+        )
```

Comment:
Why is it adding parens here? Is it a requirement or opinion?

Reply:
This is all the formatter! I made as few changes as it would let me get away with. 

The general heuristic it seems to be using is that it would rather break up the message---that is, spill onto multiple lines---for the assert rather than the assert condition. This is different from what `black` wants and is a deliberate choice they've made.

https://docs.astral.sh/ruff/formatter/black/#assert-statements

Reply:
But yeah, as far as I can tell, this is non-optional. That said, while I was initially thrown, I think I now buy the reasoning.